theme.css

```diff
@@ -4136,6 +4136,7 @@ body.anp-card-layout .workspace-tab-header-container, body.anp-card-layout .work
 body.anp-card-layout .mod-vertical .workspace-tabs {
   padding-left: var(--anp-card-layout-padding, 10px);
   padding-right: var(--anp-card-layout-padding, 10px);
+	padding-bottom: var(--anp-card-layout-padding, 10px);
```

Comment:
I believe [this code](https://github.com/AnubisNekhet/AnuPpuccin/blob/01073f95d5d213016ebfbd98a529b827f24df3d3/src/modules/Workspace/Layouts/cards.scss#L100-L106) achieves this as of now, however I am planning to refactor this segment for better organisation.